### ISSUE TITLE
Buttons stay highlighted after cursor leaves

### ISSUE THREAD

**Issue by aaronnoyes** [yeyushengfan258/Win11OS-kde#11]
When the cursor goes over the window buttons, they stay highlighted unless the cursor leaves the window. The buttons should not be highlighted once the cursor is back inside the window.

[Kazam_screencast_00000.webm](https://user-images.githubusercontent.com/30840033/198865330-4143f71d-fbaf-4cc3-a7fe-5306b00d87a6.webm)

This issue does not happen with any of the other themes that I have installed.

**Comment by aaronnoyes** [comment]
I believe that this is related to Issue #12, because when the icon size is set to Tiny, this does not happen. I'm guessing that the buttons are getting cut off, so moving the cursor down from them does not register properly. It also does not happen with maximized windows.

**Comment by yeyushengfan258** [comment]
Yes, the same problem, I will see how to fix it, fortunately it does not affect the use.

Thank you.
